code/modules/mob/living/say.dm

```diff
@@ -128,15 +128,17 @@ proc/get_radio_key_from_channel(var/channel)
 	return verb
 
 /mob/living/say(var/message, var/datum/language/speaking = null, var/verb="says", var/alt_name="")
+
 	if(client)
 		if(client.prefs.muted & MUTE_IC)
 			src << "\red You cannot speak in IC (Muted)."
 			return
 
 	if(stat)
-		if(stat == 2)
+		if(stat == DEAD)
 			return say_dead(message)
-		return
+		else if(stat != STASIS)

```

Comment:
Someone in STASIS can speak? It used to be that everyone in stasis was unconscious, and it makes sense, as by definition, your brain is shut down while you're in stasis.